sibyl/resources/model.py

```diff
@@ -283,12 +286,19 @@ def numpy_decoder(obj):
         attr_info = [
             Attrs("eids", type=None),
             Attrs("model_id"),
-            Attrs("row_id", False),
+            Attrs("row_ids", type=None, required=False),
         ]
 
-        eids, model_id, row_id = get_and_validate_params(attr_info)
-
-        entities = get_entities_table(eids, row_id)
+        eids, model_id, row_ids = get_and_validate_params(attr_info)
+        if row_ids is None:
+            entities = get_entities_table(eids, row_ids)
+        elif len(eids) > 1 and len(row_ids) > 1:
+            LOGGER.exception("Only one of eids and row_ids can have more than one element")
+            return {"message": "Only one of eids and row_ids can have more than one element"}, 400
+        elif len(eids) > 1:
+            entities = get_entities_table(eids, row_ids)
+        else:
+            entities = get_entities_table(eids, row_ids)
```

Comment:
All three conditions except the error check seem to be the same, you can just do the error check and then call get_entities_table once

Reply:
Make sense, I moved the error check inside get_entities_table